04.bowling/bowling.rb

```diff
@@ -1,10 +1,11 @@
 # frozen_string_literal: true
 
+LASTFRAME = 18
```

Comment:
lastframeという単語はないので、`_`で区切りましょう。

```suggestion
LAST_FRAME = 18
```